recipes/itypes/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "itypes" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "c6e77bb9fd68a4bfeb9d958fea421802282451a25bac4913ec94db82a899c073" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - flake8
+    - pytest
+  imports:
+    - itypes
+
+about:
+  home: https://github.com/tomchristie/itypes
+  license: BSD
```

Comment:
It would be good to have a licence_file (maybe open an issue) and a license_family

Reply:
Also 3 clause vs 2 clause spec would be good too